qiskit_nature/operators/second_quantization/qubit_converter.py

```diff
@@ -69,6 +73,11 @@ def __init__(self,
                 sector can be provided (a list of int of values -1, and 1). The default is None
                 meaning no symmetry reduction is done.
         """
+        warnings.warn('This QubitConverter is deprecated as of 0.1.0, '
```

Comment:
Aren't we already at `0.2.0` right now? I think this should be either `0.2.1` or `0.3.0`, probably the latter.

Reply:
Master is at 0.2.0 but stable is at 0.1.0. The intent is that this is fixed asap (the QubitConvertor location to avoid cycles) in 0.1.1. As I mentioned in another comment above I have changed every where we used it so its just a question of did anyone start using this in the short time since 0.1.0 came out. If they did they will now find its deprecated. I could make it 0.1.1 but we would not normally deprecate on bug fix releases - of course having something deprecated from the initial release is weird too, but I am hoping its not noticed too much given the short timeframe.